dipy/tracking/local_tracking.py

```diff
@@ -81,6 +86,9 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
             raise ValueError("step_size must be greater than 0.")
         if maxlen < 1:
             raise ValueError("maxlen must be greater than 0.")
+        if minlen >= maxlen:
```

Comment:
I would consider being a bit more strict here and raising an error in this case. If the user is asking for minlen that is larger than maxlen they must be confused about something, but I don't think that we can assume that what they want is `maxlen = minlen + 2`

Reply:
I am ok with this plan 👍, let raise an error